Sources/Extension/Alert+BSImagePicker.swift

```diff
@@ -0,0 +1,112 @@
+        alert.addAction(notNowAction)
+        
+        let openSettingsAction = UIAlertAction(title: openSettinsgLocalizedText, style: .default) { [unowned self] (_) in
+            // Open app privacy settings
+            gotoAppPrivacySettings()
+        }
+        alert.addAction(openSettingsAction)
+        present(alert, animated: true, completion: nil)
+    }
+    
+    internal func showAlerForLimitedAccess(){
+        
+        let localizedTitle = NSLocalizedString("bsimagepicker.limitedAccess.alert.title",
+                                               value: " ",
+                                               comment: "Title of Alert controller")
+        
+        let localizedMessage = NSLocalizedString("bsimagepicker.limitedAccess.alert.message",
+                                                 value: "Select more photos or go to Settings to allow access to all photos.",
+                                                 comment: "Message of Alert controller")
+        
+        let selectMorePhotosLocalizedText = NSLocalizedString("bsimagepicker.limitedAccess.alert.selectMorePhotos",
+                                                              value: "Select more photos",
+                                                              comment: "Select more photos action title")
+        
+        let allowAccessToAllPhotosLocalizedText = NSLocalizedString("bsimagepicker.limitedAccess.alert.allowAccessToAllPhotos",
+                                                                    value: "Allow access to all photos",
+                                                                    comment: "Allow access to all photos action title")
+        
+        let cancelLocalizedText = NSLocalizedString("bsimagepicker.cancel",
+                                                    value: "Cancel",
+                                                    comment: "Cancel action title")
+        
+        let actionSheet = UIAlertController(title: localizedTitle, message: localizedMessage, preferredStyle: .actionSheet)
+        
+        let selectPhotosAction = UIAlertAction(title: selectMorePhotosLocalizedText, style: .default) { [unowned self] (_) in
+            // Show limited library picker
+            if #available(iOS 14, *) {
+                PHPhotoLibrary.shared().presentLimitedLibraryPicker(from: self)
+            }
+        }
+        actionSheet.addAction(selectPhotosAction)
+        
+        let allowFullAccessAction = UIAlertAction(title: allowAccessToAllPhotosLocalizedText, style: .default) { [unowned self] (_) in
+            // Open app privacy settings
+            self.gotoAppPrivacySettings()
+        }
+        actionSheet.addAction(allowFullAccessAction)
+        
+        let cancelAction = UIAlertAction(title: cancelLocalizedText, style: .cancel, handler: nil)
+        actionSheet.addAction(cancelAction)
+        
+        present(actionSheet, animated: true, completion: nil)
```

Comment:
I get crash on iPad because the popoverPresentationController?.sourceView is not set. Send here the "manage" button so sheet is displayed properly